pvlib/tests/test_irradiance.py

```diff
@@ -286,6 +286,24 @@ def test_perez_driesse(irrad_data, ephem_data, dni_et, relative_airmass):
     assert_series_equal(out, expected, check_less_precise=2)
 
 
+def test_muneer(irrad_data, ephem_data, dni_et):
+    projection_ratio = np.array(
+        [0, 0, 0.86399, 0.169725])
+    out = irradiance.muneer(40, 180, irrad_data['dhi'], irrad_data['ghi'],
+                            dni_et, solar_zenith=ephem_data['zenith'],
+                            solar_azimuth=ephem_data['azimuth'])
+    out_Rb = irradiance.muneer(40, 180, irrad_data['dhi'], irrad_data['ghi'],
+                               dni_et,
+                               solar_zenith=ephem_data['zenith'],
+                               solar_azimuth=ephem_data['azimuth'],
+                               projection_ratio=projection_ratio)
+    expected = pd.Series(np.array(
+        [0.,   25.036,  100.759,   31.007]),
+        index=irrad_data.index)
+    assert_series_equal(out, expected, check_less_precise=2)
+    assert_series_equal(out_Rb, expected, check_less_precise=2)
+
```

Comment:
Can you repeat the same test using `np.array` as the input type, and create one more test with input being `float`?

Reply:
@cwhanse Included in commit 5632bda1e10ce987f50d006296d9652579526af2 . Thanks!